packages/angular_devkit/build_angular/src/babel/presets/application.ts

```diff
@@ -45,7 +62,6 @@ export interface ApplicationPresetOptions {
     linkerPluginCreator: typeof import('@angular/compiler-cli/linker/babel').createEs2015LinkerPlugin;
   };
 
-  forcePresetEnv?: boolean;
```

Comment:
Note: I removed this so that we just rely on `supportedBrowsers` for better type safety (otherwise it may be undefined type-wise). I don't know what the reason for this extra option is.

Reply:
It's leftover from when it needed to downlevel to ES5 based on the typescript target.  It doesn't really serve a purpose now that everything is based on the supported browsers.